docs/en/compute/Spark.md

```diff
@@ -137,7 +162,7 @@ Alternatively you can add the properties to the Hadoop configuration file
 </configuration>
 ```
 
-After Alluxio 1.8 (not included), users can encode the Zookeeper service address
+After Alluxio 1.8 (not included), users can encode the connection details
```

Comment:
Let's say Since Alluxio 2.0 . The reason I put "After Alluxio 1.8 (not included)" is because I am not sure if there will be 1.9 or going to 2.0 directly

Reply:
This is changed by the Spark PR already, so I did not change it in this PR, thanks